### PR TITLE
Set up admin schedule layout

### LINKED ISSUE #220: Admin navbar
### Summary and Background
Admin always have a navbar which allows them to access various pages.

### User Story
As an Admin, I want a central hub containing useful links, so I can always access certain important pages.

### Solution (Core Elements)
Create a navbar to be at the top of the screen whenever a volunteer is logged in.

Left: Sistering logo which links to 'Volunteer Postings' (admin homepage) page for now. This page does not formally exist yet.

Middle: Links to 'Volunteer Postings' (admin homepage) and 'User Management' pages. Both pages do not formally exist yet.

Right: Admin profile with 'Profile' (does not exist yet) and 'Logout'.

### Mockups

Other designs: https://www.figma.com/file/tOK6AjkaRZ1NBXo191D64C/Sistering-%2F-User-Authentication-Process?node-id=390%3A1975

### Tecchnical

Create a new component called `AdminNavbar.tsx` in `frontend\src\components\pages\volunteer`.
Just implement the UI. See `VolunteerNavbar.tsx` for an example.

### PR DESCRIPTION
**Pull request by LenaNguyen**
## Ticket link

Closes #220  and the ticket for the UI layout of the admin scheduling page

## Implementation description
* Created a generic navbar component that is reused between the volunteer navbar and admin navbar
* Fiddled with the margin/padding/border values to get the layout aligned. The aspect ratio between the side panel and the rest of the page seems off, even though designs say width=400px. BTW the actual side panel component is still in a pr.
    * I haven't figured out how to fit the entire calendar to the page without scrolling yet

## Steps to test
Layout: 
1. Go to `http://localhost:3000/admin/schedule/posting/1` and check that the layout matches [designs](https://www.figma.com/file/TWJTNWjK75A6VgSisySTLe/Sistering-%2F-Admin-Experience?node-id=888%3A18932)
image

Navbar 
1. For the admin navbar go to `http://localhost:3000/admin/schedule/posting/1`. both links take you to pages that currently don't exist. They should result in a 404.
2. For the volunteer navbar, go to `http://localhost:3000/volunteer/postings` and `http://localhost:3000/volunteer/shifts`. Check that both tabs take you to the correct page. 

## What should reviewers focus on?
* How closely they match designs (ratios are currently not perfect though)

I would recommend reviewing by commit and clicking ignore whitespace changes!

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/admin/schedule/ShiftTimeHeader.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const ShiftTimeHeader: React.FC<ShiftTimeHeaderProps> = ({
       px="32px"
       py="16px"
       alignItems="flex-start"
-      borderBottom="2px"
+      borderBottom="1px"
```
Comment (LenaNguyen) [inline frontend/src/components/admin/schedule/ShiftTimeHeader.tsx:26]: This was needed to make the border thickness between the ShiftTimeHeader and the AdminScheduleVolunteerTable match designs

File: frontend/src/components/admin/schedule/AdminScheduleTable.tsx [hunk 2]
```diff
@@ -107,37 +107,39 @@ const AdminScheduleTable = ({
...
-                ))
-              ) : (
-                <AdminScheduleTableRow />
-              )}
-            </Fragment>
-          );
-        })}
-      </Tbody>
-    </Table>
+    <Box borderWidth="1px" borderColor="background.dark" width="100%">
```
Comment (LenaNguyen) [inline frontend/src/components/admin/schedule/AdminScheduleTable.tsx:110]: Apply the border to the Table component directly wasn't working. Also, the border radius is off but I can't seem to get it to apply here.

File: frontend/src/components/pages/admin/schedule/AdminSchedulePostingPage.tsx [hunk 3]
```diff
@@ -161,18 +120,67 @@ const AdminSchedulePostingPage = (): React.ReactElement => {
...
+              // eslint-disable-next-line no-console
+              onShiftSelected={(shiftId: number) => console.log(shiftId)}
+            />
+            <AdminScheduleVolunteerTable
+              signups={signups}
+              numVolunteers={4}
+              selectAll={selectAllSignups}
+              updateSignupStatus={updateSignupStatus}
+            />
+          </Box>
```
Comment (LenaNguyen) [inline frontend/src/components/pages/admin/schedule/AdminSchedulePostingPage.tsx:172]: Ignore this since it'll be replaced by Brian's Side Panel component

**Review by sherryhli — APPROVED** [review]
Code LGTM! 🚢

> How closely they match designs (ratios are currently not perfect though)

Let's merge as is and check with design on if any of the deviations are significant issues

**Comment by sherryhli** [comment]
Oh would you mind renaming the PR title? 😅

**Pull request merged**